prow/plugins/trigger/pr.go

```diff
@@ -57,6 +58,10 @@ func handlePR(c client, trustedOrg string, pr github.PullRequestEvent) error {
 			return fmt.Errorf("could not validate PR: %s", err)
 		} else if trusted {
 			c.Logger.Info("Starting all jobs for updated PR.")
+			err = clearStaleComments(c.GitHubClient, trustedOrg, pr, nil)
```

Comment:
`trustedPullRequest` and `clearStaleComments` will both list the issue comments right now. Please list the issue comments once and pass them to these two functions. Then this change won't increase API token usage except for the actual comment deletion if it is needed.

Reply:
Done. Your concern on reducing API token usage always makes sense :+1: